src/models.ts

```diff
@@ -144,7 +142,9 @@ export interface IAdvancedFilterCondition {
   operator: AdvancedFilterConditionOperators;
 }
 
-export interface IAdvancedFilter extends IFilter {
+export interface IAdvancedFilter {
+  $schema: string;
+  target: IFilterTarget;
```

Comment:
Why copy these properties instead of extending an interface. They are shared and the interface should enforce this.